docs/local-dev/debugging.md

```diff
@@ -71,3 +71,17 @@ In your `launch.json`, add a new entry with the following,
     "console": "integratedTerminal"
 },
 ```
+
+### WebStorm
+
+Select `Edit Configurations` in the `Run` dropdown menu. Click the plus sign to add a new configuration, then select `Node.js`.
+
+In `Working directory`, input `${workspaceFolder}/packages/backend`.
+
+In `JavaScript file`, input `${workspaceFolder}/node_modules/@backstage/cli/bin/backstage-cli`.
```

Comment:
Is `${workspaceFolder}` a common convention for WebStorm? I don't use it but want to make sure this would be clear to someone who does.

Reply:
I honestly just borrowed it from the above VSCode section, not sure what a more generic one would be but I could try it

Reply:
It's an actual VSCode variable: https://code.visualstudio.com/docs/editor/variables-reference#_predefined-variables

Reply:
But that works in WebStorm? If so I'm fine with that